ansible/roles_ocp_workloads/ocp4_workload_quarkus_workshop_user/tasks/pre_workload.yml

```diff
@@ -0,0 +1,84 @@
+---
+# Implement your Pre Workload deployment tasks here
+
+- name: Ensuring username is defined
+  fail:
+    msg: "This workload requires ocp_username to be defined. Exiting..."
+  when: ocp_username is not defined
+
+- name: Ensuring guid is defined
+  fail:
+    msg: "This workload requires guid to be defined. Exiting..."
+  when: guid is not defined
+
+# Figure out paths
+- name: extract api_url
+  command: oc whoami --show-server
```

Comment:
https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles_ocp_workloads/ocp4_workload_rhtr_pipelines_user/tasks/workload.yml#L96-L131 shows a more native way of doing this.

Reply:
Done! :)